orchestration/scripts/diff-hca-table.sh

```diff
@@ -0,0 +1,49 @@
+set -euo pipefail
+
+# Copy-pasted from StackOverflow.
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+# Point to BQ metadata we expect to be present on disk.
+declare -r TABLE_DIR=/bq-metadata/${TABLE}
+declare -r PK_COLS=$(cat ${TABLE_DIR}/primary-keys)
+declare -r COMPARE_COLS=$(cat ${TABLE_DIR}/compare-cols)
+
+declare -a DATAREPO_PKS=()
+for col in ${PK_COLS//,/ }; do
+  DATAREPO_PKS+=("${col} as datarepo_${col}")
+done
+declare -r REPO_KEYS=$(join_by ', ' "${DATAREPO_PKS[@]}")
```

Comment:
Note that when you use `declare` on a subshell result you're throwing away the return code of the subshell, and `set -e` won't get triggered if the subshell return code is nonzero. In this case it doesn't matter but it can lead to problems if you're not careful.

For more info see https://github.com/koalaman/shellcheck/wiki/SC2155